user profile/user_profile.css

```diff
@@ -0,0 +1,111 @@
+.navbar{
+    padding: 0.25rem;
+ }
+ .search-container{
+    flex-basis: 25%;
+    border-radius: 1.5rem;
+    background: var(--white)
+ } 
+.search-bar{
+    outline: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    padding: 0.6rem 0.8rem;
+    margin: 0.5rem;
+}
+/* user profile styling */
+.user-info{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+/* address styling */
+.slider-input-titles{
+    margin-top: 3rem;
+}
+.wrapper{
+    max-width: 50%;
+    width: 100%;
+    margin: 8rem auto;
+    padding: 1rem;
+    background: var(--white);
+    box-shadow: 0 0 3px 3px var(--card-border);
+}
+.wrapper header{
+    font-size: 2rem;
+    font-weight: 600;
+    letter-spacing: 1px;
+    margin-bottom: 5rem;
+    text-align: center;
+}
+.wrapper nav{
+    position: relative;
+    height: 50px;
+    width:80%;
+    display: flex;
+    align-items: center;
+}
+#home:checked ~ nav label.home,
+#product:checked ~ nav label.product,
+#help:checked ~ nav label.help,
+#contact:checked ~ nav label.contact
+{
+    color: var(--white);
+}
+.wrapper nav label{
+    display: block;
+    height: 100%;
+    width:100%;
+    text-align: center;
+    cursor: pointer;
+    line-height: 50px;
+    color:var(--primary-light);
+    font-family: 17px;
```

Comment:
what's this font-family 17px used for!!

Reply:
It's not needed,  will remove that. Thank you for bringing it to my notice.